srcDoc/release_notes.rst

```diff
@@ -54,6 +54,17 @@ What's New in Master
 What's Fixed in Master
 ~~~~~~~~~~~~~~~~~~~~~~
 
+The version of *rtiddsgen* is now properly compared to identify the support of certain features (#237)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The line of `^` has to coincide with the length of the tittle. The tittle doesn't need to say that it is fixed. Also, put the documentation in order wrt the number of the issue.

"Issue obtaining and comparing the `rtiddsgen` version in the compilation scripts"

Reply:
Done. I miss that.